### DISCUSSION TITLE
[功能建议]动态首页中的时间信息

### DISCUSSION THREAD

**Discussion by NightSharp** [the1812/Bilibili-Evolved#2348] [功能建议 / Ideas]
希望能够在动态首页的动态中显示一天之前的动态的具体时间信息(精确到小时和分钟)，而不是只显示一个日期

**Comment by Geeyun-JY3** [comment]
很遗憾：#839。

不过 BilPlus 是可以拿到的，可以跳转 BiliPlus 后找对应动态。

**Reply by NightSharp** [reply]
> 
> 
> 很遗憾：#839。
> 
> 不过 BilPlus 是可以拿到的，可以跳转 BiliPlus 后找对应动态。

这个跟评论区的情况不一样吧，像是一天以内的动态，昵称下方显示的是精确到分钟的，超过一天后就只有日期了
包括动态单项和动态首页
另外可能是我装了另一个脚本的缘故，我查看的页面的评论区显示的都是精确到分钟的时间

**Reply by Geeyun-JY3** [reply]
> 这个跟评论区的情况不一样吧，像是一天以内的动态，昵称下方显示的是精确到分钟的，超过一天后就只有日期了
> 包括动态单项和动态首页

对啊 #839 就是希望能显示动态的绝对时间啊。
为啥提到了评论区……你想要的不是动态昵称下方显示的时间吗？

> 另外可能是我装了另一个脚本的缘故，我查看的页面的评论区显示的都是精确到分钟的时间

并不太懂你这句话说的什么意思，建议指出是哪个脚本。

**Reply by NightSharp** [reply]
> 
> 
> > 这个跟评论区的情况不一样吧，像是一天以内的动态，昵称下方显示的是精确到分钟的，超过一天后就只有日期了
> > 包括动态单项和动态首页
> 
> 对啊 #839 就是希望能显示动态的绝对时间啊。 为啥提到了评论区……你想要的不是动态昵称下方显示的时间吗？
> 
> > 另外可能是我装了另一个脚本的缘故，我查看的页面的评论区显示的都是精确到分钟的时间
> 
> 并不太懂你这句话说的什么意思，建议指出是哪个脚本。

可能是我看岔了

脚本的话是这个
[Bilibili 翻页评论区](https://greasyfork.org/zh-CN/scripts/403348-bilibili-翻页评论区)

**Reply by Geeyun-JY3** [reply]
> 脚本的话是这个
> [Bilibili 翻页评论区](https://greasyfork.org/zh-CN/scripts/403348-bilibili-%E7%BF%BB%E9%A1%B5%E8%AF%84%E8%AE%BA%E5%8C%BA)

这个脚本我也在用，显示的时间和原版没区别的呀。

**Reply by NightSharp** [reply]
> 
> 
> > 脚本的话是这个
> > [Bilibili 翻页评论区](https://greasyfork.org/zh-CN/scripts/403348-bilibili-%E7%BF%BB%E9%A1%B5%E8%AF%84%E8%AE%BA%E5%8C%BA)
> 
> 这个脚本我也在用，显示的时间和原版没区别的呀。

动态时间超过一天后就不显示时间和分钟了只有日期，但这个脚本下的评论区不存在这个限制

**Reply by Geeyun-JY3** [reply]
动态时间是动态时间，评论区是评论区，本来就是不一样的机制，和用不用这个脚本没关系。

我也想动态时间能和评论区一样，可惜 #839 已经明说了做不到。

**Reply by NightSharp** [reply]
> 
> 
> 动态时间是动态时间，评论区是评论区，本来就是不一样的机制，和用不用这个脚本没关系。
> 
> 我也想动态时间能和评论区一样，可惜 #839 已经明说了做不到。

好吧，看来只能另想办法了
